gensim/test/test_similarities.py

```diff
@@ -371,6 +372,7 @@ def testIter(self):
 
 
 class TestSoftCosineSimilarity(unittest.TestCase, _TestSimilarityABC):
+    @deprecated("Method will be removed in 4.0.0")
```

Comment:
I don't think that this is a good idea to deprecate something in tests. Anyway, if we'll remove old code - tests will be broken and we'll see it clearly.

Reply:
After we have decided what to actually deprecate, I will remove the annotations from tests. They are currently there just to make it clear what parts of the code are proposed for deprecation.